sdf/final-project/business.md

```diff
@@ -1,56 +1,49 @@
 # SDF Final Project Rubric - Business
-
 This is intended to be filled out by staff members of DPI Tech Apprenticeship program.
 
-Date/Time:
-
-Trainee Name:
-
-Project Name:
-
-Reviewer Name:
+- Date/Time:
+- Trainee Name:
+- Project Name:
+- Reviewer Name:
 
-## Requirements
+## Functional Requirements Document
 Should have a functional specification document with:
-
-    [ ] a description of the problem
-
-    [ ] proposed solution
-
-    [ ] user stories
-
-    [ ] user interface / flow sketches
-
-[ ] Does the app satisfy the functional requirements?
+- [ ] description of the problem
+- [ ] proposed solution
+- [ ] user stories
+- [ ] domain model, associations
```

Comment:
Can we specify that they should paste in a copy/screenshot using the ERD tool?

Reply:
ah, I think either pseudocode, docs or a screenshot are all fine.

Reply:
Sounds good!